README.md

```diff
@@ -2,15 +2,88 @@
 
 Ceci est un benchmark [Steampipe][] pour vérifier le respect de quelques règles pour nos applications hébergées par [Scalingo][].
 
-Pour jouer ces règles:
-1. Installer [Steampipe][]
-1. Installer et configurer le plugin scalingo: `steampipe plugin install francois2metz/scalingo`
-1. Cloner ce dépot
-1. Lancer la commande: `steampipe check benchmark.scalingo`
+## Installation
 
-Pour exclure certaines applications de la vérification des logs routeurs:
+### Steampipe
```

Comment:
Est-ce qu'on peut déplacer tout ce qui n'est pas en rapport avec le problème qu'on résout sur une autre PR ?

Reply:
Be my guest

Reply:
https://github.com/1024pix/pix-steampipe-mod-scalingo-compliance/pull/9